tests/plugins/test_manager.py

```diff
@@ -31,18 +33,18 @@ def conda_solvers(self):
 
 def test_load_no_plugins(plugin_manager):
     plugin_names = plugin_manager.load_plugins()
-    assert plugin_names == []
+    assert not plugin_names
```

Comment:
I think this assertion statement would be a little clearer if we changed it to:

```suggestion
    assert plugin_names == 0
```

It would also be more consistent with the assertions in the tests below.

Reply:
Agreed